pkg/bindinfo/binding_cache.go

```diff
@@ -112,11 +140,22 @@ func (fbc *fuzzyBindingCache) getFromMemory(sctx sessionctx.Context, fuzzyDigest
 	return matchedBinding, isMatched, missingSQLDigest
 }
 
-func (fbc *fuzzyBindingCache) loadFromStore(missingSQLDigest []string) {
+func (fbc *fuzzyBindingCache) loadFromStore(sctx sessionctx.Context, missingSQLDigest []string) {
+	if intest.InTest {
+		n := sctx.Value(LoadBindingNothing)
+		if n != nil {
```

Comment:
Could we write it more tightly: `if intest.InTest && sctx.Value(...) != nil`?

Reply:
Done